app/models/concerns/orchestration/common.rb

```diff
@@ -5,4 +5,11 @@ def handle_validation_errors
     logger.debug "Error occured during validations of #{self.class.name}: #{e.message}"
     nil
   end
+
+  def log_orchestration_errors
+    errors = []
```

Comment:
This cannot be named `errors` as this shadows the ActiveRecord `errors` collection. Please rename this to `my_errors` or `orchestration_errors` or whatever...